ddl/stat_test.go

```diff
@@ -87,9 +87,7 @@ LOOP:
 	for {
 		select {
 		case <-ticker.C:
-			d.Stop()
 			c.Assert(s.getDDLSchemaVer(c, d), GreaterEqual, ver)
-			d.restartWorkers(context.Background())
```

Comment:
Why remove this test? Will deleting this test reduce test scenarios?

Reply:
`d.Stop()`will call `d.cancel()`, which will trigger `doDDLJob` quit

Reply:
We can replace `done <- d.doDDLJob(ctx, job)` with checking the job in history. Instead of removing these test scenarios.